R/accounts.R

```diff
@@ -60,74 +60,121 @@ connectApiUser <- function(account = NULL, server = NULL, apiKey, quiet = FALSE)
 
   registerAccount(
     serverName = server,
-    accountName = account %||% user$username,
+    accountName = user$username,
     accountId = user$id,
     apiKey = apiKey
   )
 
   if (!quiet) {
-    message("Account registered successfully: ", account %||% user$username)
+    accountId <- accountId(user$username, server)
```

Comment:
nit: There are two notions of `accountId`: the user identifier from the API, and `username@server`. That's bound to confuse our future selves.

Reply:
Yeah, I'll brainstorm a better name.

Reply:
I have failed to come up with a better name; but I've made a note to keep thinking about it.